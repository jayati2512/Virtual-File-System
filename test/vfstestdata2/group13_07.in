createvfs iiitb 1024
mountvfs iiitb
makedir /country
makedir /account
makedir /country india
makedir /country america
makedir /account checkingacc
makedir /account savingacc
makedir /account australia
movedir /account/australia /country
addfile /account/checkingacc acc.txt /user/include
addfile /country/america ssn.txt /user/include
listdir /country 5 /tmp
listdir /country 0 /tmp
addfile /country/savingacc hello.c /user/include
listfile /country/savingacc/hello.c /tmp
updatefile /country/america/ssn.txt /user/include
exportfile /country/savingacc/hello.c /tmp/abc.c
copyfile /country/america/ssn.txt
removefile /country/america/ssn.txt
searchfile ss /tmp
unmountvfs iiitb
deletedir /country/australia
