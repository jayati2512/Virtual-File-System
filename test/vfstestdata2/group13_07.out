createvfs_SUCCESS
mountvfs_SUCCESS
ERR_VFS_MAKEDIR: VFS_INSUFFICIENT_ARGUMENTS
ERR_VFS_MAKEDIR: VFS_INSUFFICIENT_ARGUMENTS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
movedir_SUCCESS
ERR_VFS_ADDFILE:UNABLE_TO_OPEN_SPECIFIED_DATAFILE_PATH
ERR_VFS_ADDFILE:UNABLE_TO_OPEN_SPECIFIED_DATAFILE_PATH
ERR_VFS_LISTDIR: INVALID_FILE_NAME
ERR_VFS_LISTDIR: INVALID_FILE_NAME
ERR_VFS_ADDFILE: UNABLE_TO_CREATE_NEWFILE
ERR_VFS_LISTFILE: SOURCE_FILE_PATH_NOT_FOUND
ERR_VFS_UPDATEFILE: SOURCE_FILE_PATH_NOT_FOUND
ERR_VFS_EXPORTFILE: CANNOT_FIND_SPECIFIED_SOURCEFILE
ERR_VFS_COPYFILE: VFS_INSUFFICIENT_ARGUMENTS
ERR_VFS_REMOVEFILE: CANNOT_FIND_SPECIFIED_FILE
ERR_VFS_SEARCHFILE:FILE_TO_BE_SEARCHED_NOT_FOUND
unmountvfs_SUCCESS
ERR_VFS_DELETEDIR: VFS_NOT_MOUNTED
